common/persistence/serialization/interfaces.go

```diff
@@ -285,6 +285,7 @@ type (
 	// TimerTaskInfo blob in a serialization agnostic format
 	TimerTaskInfo struct {
 		DomainID        UUID
+		Domain          string
```

Comment:
Could you revert this change? This is not necessary

Reply:
Done. Do you want me to add any other change as well before i push it again?